packages/snaps-sdk/src/types/methods/get-preferences.ts

```diff
@@ -13,9 +13,23 @@ export type GetPreferencesParams = never;
  * @property locale - The user's selected locale.
  * @property currency - The user's selected currency.
  * @property hideBalances - Whether the user has chosen to hide balances.
+ * @property useSecurityAlerts - Whether to run transactions and signatures through blockaid.
+ * @property useSimulations - Whether to simulate transactions and signatures.
+ * @property useTokenDetection - Whether to auto-detect tokens.
+ * @property batchCheckBalances - Whether to fetch balances in an aggregated manner.
+ * @property displayNftMedia - Whether to display NFT media.
+ * @property useNftDetection - Whether to auto-detect NFTs.
+ * @property usePriceCheck - Whether to get token price data from CoinGecko and CryptoCompare.
  */
 export type GetPreferencesResult = {
   locale: string;
   currency: string;
   hideBalances: boolean;
+  useSecurityAlerts: boolean;
+  useSimulations: boolean;
```

Comment:
```suggestion
  useTransactionSimulations: boolean;
```

Reply:
It can be signatures as well, so that's why I kept it generic.

Reply:
Hmm, I feel like "simulations" is _too_ generic, but I'm not sure if there's a good name for both 🤔

Reply:
How about `simulateOnChainActions`?

Reply:
```suggestion
  useOutcomeSimulations: boolean;
```
Perhaps?

Reply:
More or less ambigious than `useSimulations`? 😄

Reply:
well i like `simulateOnChainActions` a bit better lol

Reply:
Done.